placehero/src/lib.rs

```diff
@@ -36,54 +58,178 @@ impl Placehero {
                 prose(instance.title.as_str()),
             )))
         } else {
-            OneOf::B(label("Not yet connected (or other unhandled error)"))
+            OneOf::B(prose("Not yet connected (or other unhandled error)"))
+        }
+    }
+
+    fn main_view(&mut self) -> impl WidgetView<Self> + use<> {
+        if self.statuses.is_empty() {
+            OneOf::A(prose("No statuses yet loaded"))
+        } else {
+            Either::B(portal(flex(
```

Comment:
nit: mixing `Either` and `OneOf` looks weird to me. (same above)

Reply:
I think I agree; this format is idiomatic for `OneOf3` and above (it means that if you add a new variant, you only need to update the count on the first), but for `Either`, I think it is right to use `Either` for both.